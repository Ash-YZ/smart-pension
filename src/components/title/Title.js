import React from 'react'
import PropTypes from 'prop-types'
import './styles.scss'

const Title = props => {

  return (
    <div className='container'>
      <div className='row'>
        <div className='col-12'>
          {props.isMain ?
            <h1>{props.title}</h1> :
            <h2>{props.title}</h2>
          }
        </div>
      </div>
    </div>
  )
}

Title.propTypes = {
  title: PropTypes.string,
  isMain: PropTypes.bool
}

export default Title