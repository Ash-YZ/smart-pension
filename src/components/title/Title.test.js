import React from 'react';
import { shallow } from 'enzyme'
import Title from './Title'

describe('Title', () => {
  it('should render main title correctly ', () => {
    const component = shallow(<Title title='Test Title' isMain />)
    expect(component).toMatchSnapshot()
  })

  it('should render sub-title correctly ', () => {
    const component = shallow(<Title title='Test Title' />)
    expect(component).toMatchSnapshot()
  })
})