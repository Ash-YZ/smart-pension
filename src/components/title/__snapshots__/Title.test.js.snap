// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render main title correctly  1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-12"
    >
      <h1>
        Test Title
      </h1>
    </div>
  </div>
</div>
`;

exports[`Title should render sub-title correctly  1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-12"
    >
      <h2>
        Test Title
      </h2>
    </div>
  </div>
</div>
`;
